core/src/main/java/org/frankframework/filesystem/SftpFileSystem.java

```diff
@@ -40,14 +41,18 @@
 import com.jcraft.jsch.SftpATTRS;
 import com.jcraft.jsch.SftpException;
 
+import lombok.AccessLevel;
 import lombok.Getter;
+import lombok.Setter;
 
 /**
  * Implementation of SFTP FileSystem
  *
  * @author Niels Meijer
  */
 public class SftpFileSystem extends SftpSession implements IWritableFileSystem<SftpFileRef> {
+	private static final long RECHECK_CONNECTION_INTERVAL_SECONDS = 60;
```

Comment:
Is it a good idea to make this configurable?

Reply:
Yeah, I thought about that and I've asked that to Niels too and we thought that's not needed. Next time, I'll discuss that in the other app group directly, to shortcut this faster.